models/hospitalAdministrator.js

```diff
@@ -8,8 +10,14 @@ module.exports = (sequelize, DataTypes) => {
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
-      // define association here
-      // TODO
+      HospitalAdministrator.belongsTo(Hospital, {
```

Comment:
This is correct, with a change: instead of just ```Hospital```, reference ```models.Hospital```, from the models object argument passed in to the method.

Then you can DELETE the require statement on line 3.

Models _shouldn't/can't_ require other models- the reason is that we'll likely end up in a circular reference loop that will cause execution to fail.

So, instead the index.js handles loading and setting things up in two passes- first it will load ALL the model files in the directory, collecting all the model classes into one object, THEN call the associate function on _each_ model, passing in the object with all of the models.

Reply:
Ah, ok. Now that comment block above `associate` makes sense 😆